blackbird/utils/configread.py

```diff
@@ -661,7 +663,8 @@ def is_log(value):
 
            Recommended giving he full path including the file name.
 
-    :param str value: log file path. You can specify not only absolute path but also relational path.
+    :param str value: log file path. You can specify not only absolute path
+           but also relational path.
```

Comment:
Got it. I'd like to style 2 too.
btw, I suggest style 3.

### 3.
```
:param str value:
    log file path
    You can specify not only absolute path but also relational path.
```
How do you think?